.github/sync-translations.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+// This script should be run after adding a new message to translate.
+// It will ensure that all messages in "*.en.xlf" files are propagated to all languages.
+// The resulting diff should then be submitted as a PR on the lowest maintained branch,
+// possibly after using GPT to translate all the targets it contains
+// (state="needs-review-translation" should then be used on corresponding target tags.)
+
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Translation\Loader\XliffFileLoader;
+use Symfony\Component\Translation\MessageCatalogue;
+
+require __DIR__.'/../vendor/autoload.php';
+
+function dumpXliff1(string $defaultLocale, MessageCatalogue $messages, string $domain)
+{
+    $dom = new \DOMDocument('1.0', 'utf-8');
+    $dom->formatOutput = true;
+
+    $xliff = $dom->appendChild($dom->createElement('xliff'));
+    $xliff->setAttribute('version', '1.2');
+    $xliff->setAttribute('xmlns', 'urn:oasis:names:tc:xliff:document:1.2');
+
+    $xliffFile = $xliff->appendChild($dom->createElement('file'));
+    $xliffFile->setAttribute('source-language', str_replace('_', '-', $defaultLocale));
+    $xliffFile->setAttribute('target-language', 'no' === $messages->getLocale() ? 'nb' : str_replace('_', '-', $messages->getLocale()));
```

Comment:
why this `no` vs `nb` thing ? Why would we have a `no` locale in Symfony if that's not the actual code of the locale ?

Reply:
we have a test case that ensures that "no" is an alias of "nb", which it is semantically.